app/main.py

```diff
@@ -1,8 +1,32 @@
 class Person:
-    # write your code here
-    pass
+    people = {}  # Атрибут класса для хранения экземпляров по их имени
 
+    def __init__(self, name: str, age: int):
+        self.name = name
```

Comment:
The `Person` class should have attributes for `wife` and `husband` to store spouse references. Consider initializing these attributes in the constructor.